src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -782,26 +784,51 @@ public Response getPolicy(@PathParam("id") UUID policyId) {
 
   @Operation(summary = "Retrieve the trigger history of a single policy")
   @APIResponse(responseCode = "200", description = "History could be retrieved",
-      content = @Content (schema = @Schema(type = SchemaType.ARRAY, implementation = HistoryItem.class)))
+      content = @Content (schema = @Schema(implementation = PagingUtils.PagedResponse.class)),
```

Comment:
I have a problem here, using this syntax removes the `HistoryItem` from the openapi.json plus the `PagedResponse` doesn't have any info about its contents.

I found the following [1] workaround:
```java
// workaround for returning generic types: https://github.com/swagger-api/swagger-core/issues/498#issuecomment-74510379
  // This class is used only for swagger return type
  private static class PagedResponseOfHistoryItem extends PagingUtils.PagedResponse<HistoryItem> {
    private List<HistoryItem> data;
    private PagedResponseOfHistoryItem() {}
  }

  @Operation(summary = "Retrieve the trigger history of a single policy")
  @APIResponse(responseCode = "200", description = "History could be retrieved",
      content = @Content (schema = @Schema(implementation = PagedResponseOfHistoryItem.class)),
```

WDYT?

[1] https://github.com/swagger-api/swagger-core/issues/498#issuecomment-74510379

Reply:
I was thinking about something similar, as I don't like the type-erasure. 
Same for `getPolicies`